frontend/src/components/AboutMe/AboutMe.jsx

```
@@ -23,71 +27,42 @@ const UserProfile = () => {
-            </p>
-            <p className="flex items-center gap-2 text-gray-700">
-              <Building2 className="w-5 h-5 text-blue-500" />
-              <span><strong>Status:</strong> {user.is_active ? "Active" : "Inactive"}</span>
-            </p>
-          </div>
+      
+      {/* Middle Section */}
+      <div className="w-full md:w-2/4 px-4 mt-20 md:mt-24 -mb-24">
+        <div className="flex justify-around border-b pb-2">
+          <button className={`px-4 py-2 ${activeTab === "posts" ? "font-bold" : "text-gray-600"}`} onClick={() => setActiveTab("posts")}>Posts</button>
+          <button className={`px-4 py-2 ${activeTab === "research" ? "font-bold" : "text-gray-600"}`} onClick={() => setActiveTab("research")}>Research</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)